ludwig/config_validation/checks.py

```diff
@@ -374,3 +374,36 @@ def is_divisible(hidden_size: int, num_heads: int) -> bool:
                 f"Input feature {input_feature.name} transformer encoder requires encoder.hidden_size to be divisible "
                 f"by encoder.num_heads. Found hidden_size {encoder.hidden_size} and num_heads {encoder.num_heads}."
             )
+
+
+@register_config_check
+def check_tagger_decoder_requirements(config: "ModelConfig") -> None:  # noqa: F821
+    """Checks that the tagger decoder has at least one sequence, text or timeseries input feature where the
+    encoder's reduce_output will produce a 3D shaped output from the combiner."""
+    # Check if there is a text or sequence output feature using a tagger decoder
+    output_feature_with_tagger_decoder = False
+    for output_feature in config.output_features:
+        if output_feature.type in {TEXT, SEQUENCE} and output_feature.decoder.type == "tagger":
+            output_feature_with_tagger_decoder = True
+
+    if not output_feature_with_tagger_decoder:
+        return
+
+    # Check that there is at least one sequence, text or timeseries input feature that doesn't reduce the
+    # output of the encoder.
+    one_feature_with_reduce_output_none = False
```

Comment:
nit: Can we make do without this additional boolean?

Reply:
Done!